javascript/extractor/lib/typescript/src/main.ts

```diff
@@ -162,10 +162,10 @@ function prepareNextFile() {
     }
 }
 
-function handleParseCommand(command: ParseCommand) {
+function handleParseCommand(command: ParseCommand, checkPending = true) {
```

Comment:
This change is not needed to support `import.meta`.  
But it was required for me to run the "standalone" commands, so I kept the change.